presto-product-tests/conf/tempto/tempto-configuration-for-docker-default.yaml

```diff
@@ -4,3 +4,8 @@ databases:
   presto:
     host: presto-master
     configured_hdfs_user: hive
+    cli_ldap_truststore_path: /etc/ldap/cacerts.jks
```

Comment:
Why include ldap configuration by default? I'd say let's keep the separate tempto configs in each profile's dir.

Also, this change should be a separate commit. But let's stick to tempto-config-per-environment. If we find the config duplication a problem, we can think of merging the configs in a way similar to what docker-copose does with multiple compose files.